.github/workflows/php.yml

```diff
@@ -90,6 +91,8 @@ jobs:
             php-version: 7.4
           - project: 'Instrumentation/OpenAIPHP'
             php-version: 8.0
+          - project: 'Instrumentation/CakePHP'
+            php-version: 8.0
```

Comment:
this excludes running tests against php 8.0, but composer.json says it supports 8.0. Did you mean to exclude 7.4?

Reply:
Yeah sorry. Should i exclude this entirely from here ? As we don't have any test cases setup for this currently ?

Reply:
No, it should still run even with zero tests (because there are a lot of other checks that also run that are important).

Reply:
Done